### PR TITLE
Implements reading the local_address config option

### PR DESCRIPTION
**Pull request by terwey**

### CODE REVIEW

File: cmd/shadowsocks-local/local.go [hunk 1]
```diff
@@ -355,14 +355,14 @@ func enoughOptions(config *ss.Config) bool {
...
-	var configFile, cmdServer, cmdLocal string
+	var configFile, cmdServer string
 	var cmdConfig ss.Config
 	var printVer bool
 
 	flag.BoolVar(&printVer, "version", false, "print version")
 	flag.StringVar(&configFile, "c", "config.json", "specify config file")
 	flag.StringVar(&cmdServer, "s", "", "server address")
-	flag.StringVar(&cmdLocal, "b", "", "local address, listen only to this address if specified")
+	flag.StringVar(&cmdConfig.LocalAddress, "b", "", "local address, listen only to this address if specified")
```
Comment (arthurkiller) [inline cmd/shadowsocks-local/local.go]: why we need to clearify the local address?

Comment (arthurkiller) [reply cmd/shadowsocks-local/local.go]: @terwey  that's unnecessary for us to specify the local port

**Comment by arthurkiller** [comment]
Unnecessary
WONT MERGE

**Comment by tanji** [comment]
It's necessary because your default is 0.0.0.0 which is wrong and insecure, and the `local_address` option exists in initial implementation of Shadowsocks, so your implementation isn't feature compatible because it just ignores this option.

**Comment by arthurkiller** [comment]
@tanji Listen on 0:8080 will cause nothing wrong

**Comment by tanji** [comment]
From your standpoint, maybe, but some people use shadowsocks for personal or business usage and hosting providers do not really like open proxies. Please consider this request as it does not alter current functionality, it just allows the configuration to work as intended in the original implementation.

**Comment by arthurkiller** [comment]
@tanji K, I will merge this

**Pull request merged**